client/src/main/python/slipstream/resources/reports.py

```diff
@@ -0,0 +1,162 @@
+"""
+ SlipStream Client
+ =====
+ Copyright (C) 2016 SixSq Sarl (sixsq.com)
+ =====
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+from __future__ import print_function
+
+import os
+import json
+import urllib2
+from datetime import datetime
+import pprint
+
+from slipstream.HttpClient import HttpClient
+import slipstream.util as util
+from slipstream.NodeDecorator import NodeDecorator
+
+
+def _download(username, password, src_url, dst_file):
+    request = urllib2.Request(src_url)
+    request.add_header('Authorization',
+                       (b'Basic ' + (username + b':' + password).encode('base64')).replace('\n', ''))
```

Comment:
So I think we should use cookie auth for new stuff (and this is a new command 😉)